cmd/edgex-secrets-config/README.md

```diff
@@ -0,0 +1,181 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+edgex-secrets-config-proxy – Configure EdgeX API gateway service
+
+# SYNOPSIS
+
+**edgex-secrets-config proxy** SUBCOMMAND [OPTIONS]
+
+# DESCRIPTION
+
+Configures the EdgeX API gateway service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and other related utility functions.
+
+Proxy configuration commands (listed below) require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **--confdir** _/path/to/directory/with/configuration.toml_ (optional)
```

Comment:
Should this tool support all of the standard config file command-line options?

```
d.FlagSet.StringVar(&d.configFileName, "f", DefaultConfigFile, "")
d.FlagSet.StringVar(&d.configFileName, "file", DefaultConfigFile, "")
d.FlagSet.StringVar(&d.profile, "profile", "", "")
d.FlagSet.StringVar(&d.profile, "p", "", ".")
d.FlagSet.StringVar(&d.configDir, "confdir", "", "")
d.FlagSet.StringVar(&d.configDir, "c", "", "")
```

Reply:
No reason to as it is a one-shot utility.  This is a fork of security-proxy-setup which doesn't support these flags either.

Reply:
For future, we may want to consider a reusable `securityFlags` that is common to these security utilities/services that only need --confdir. Andre's latest service really on needs --confdir also.